files/index.html

```diff
@@ -7,9 +7,11 @@
     <title>looking-glass-fusion</title>
     <style>
         html, body { margin: 0; overflow: hidden; cursor: crosshair;}
+        #VRButton {font-size: 150% !important; color: black !important;}
     </style>
+    <p>You may notice the looking glass window stops responding to user input. To avoid this do not minimize this window after starting the XHR session and do not place it underneath the XHR session</p>
```

Comment:
I think you meant "WebXR" here?
```suggestion
    <p>You may notice the looking glass window stops responding to user input. To avoid this do not minimize this window after starting the WebXR session and do not place it underneath the WebXR session</p>
```

Reply:
Hahaha old lingo dies hard. Thanks for the catch.